utils/osutil/osutil.go

```diff
@@ -176,3 +177,24 @@ func findFileInSubfolders(startDir, targetFileName string) (string, error) {
 
 	return filePath, nil
 }
+
+// FindDirectoriesContainingFile finds all directories containing a file matching the given regular expression
```

Comment:
I created similar stuff in ccip -https://github.com/smartcontractkit/ccip/blob/ccip-develop/integration-tests/ccip-tests/utils/fileutil.go
is it worth moving the entire one here?

Reply:
It probably is worthwhile to have them all in one place. Created a ticket for that work: https://smartcontract-it.atlassian.net/browse/TT-1076